package.json

```diff
@@ -10,7 +10,9 @@
     "test:e2e:android": "pushd detox/test && npm run e2e:android && popd",
     "release": "node scripts/ci.release.js",
     "package:android": "node scripts/ci.android-release.js",
-    "package:ios": "scripts/ci.ios-release.sh"
+    "package:ios": "scripts/ci.ios-release.sh",
+    "ci.ios": "sh ./scripts/ci.ios.sh",
+    "ci.android": "sh ./scripts/ci.android.sh"
```

Comment:
let's change it to `ci:ios` and `ci:android` and fix that on the docs.

Reply:
fixed